kubernetes/types.go

```diff
@@ -113,6 +113,18 @@ var (
 	}
 	ApiNetworkingVersionV1Alpha3 = NetworkingGroupVersionV1Alpha3.Group + "/" + NetworkingGroupVersionV1Alpha3.Version
 
+	NetworkingGroupVersionV1Beta1 = schema.GroupVersion{
+		Group:   "networking.istio.io",
+		Version: "v1beta1",
+	}
+	ApiNetworkingVersionV1Beta1 = NetworkingGroupVersionV1Beta1.Group + "/" + NetworkingGroupVersionV1Beta1.Version
```

Comment:
Do the older versions need to stay here?

Reply:
It is used in `kubernetes/filters.go`:

```
func FilterSupportedGateways(gateways []*networking_v1.Gateway) []*networking_v1.Gateway {
	filtered := []*networking_v1.Gateway{}
	for _, gw := range gateways {
		if gw.APIVersion == ApiNetworkingVersionV1 || gw.APIVersion == ApiNetworkingVersionV1Beta1 || gw.APIVersion == ApiNetworkingVersionV1Alpha3 {
			filtered = append(filtered, gw)
		}
	}
	return filtered
}
```
Actually I don't know why we are checking the apiVersion of the gateway. Do you think we could directly remove that if statement?
```
for _, gw := range gateways {
	filtered = append(filtered, gw)
}
```

Reply:
It seems `FilterSupportedGateways` is not used anymore, so it can be removed. 
Another place where `NetworkingGroupVersionV1Beta1` is used is https://github.com/kiali/kiali/blob/master/business/istio_config.go#L978 but it needs to be edited to `NetworkingGroupVersionV1`.
And you can remove 'NetworkingGroupVersionV1Beta1'.

Reply:
Thanks @hhovsepy, I have removed `NetworkingGroupVersionV1Beta1` with your comments.